src/shared/directives/conditional-show/conditional-show-form.directive.ts

```diff
@@ -121,18 +109,18 @@ export class ConditionalShowFormDirective implements OnInit, AfterViewInit, OnDe
   }
 
   // make sure for the 3 callbacks that we are bound to this via an arrow function
-  private handleFormControl = (c: FormControl) => {
+  private handleFormControl = (c: FormControl): void => {
     this.evaluateControl(c);
   }
 
-  private handleFormArray = (c: FormArray, caseField: CaseField) => {
-    this.evaluateControl(c);
-    c.controls.forEach((formControl, ix) => {
-      this.fieldsUtils.controlIterator(formControl, this.handleFormArray, this.handleFormGroup, this.handleFormControl)
+  private handleFormArray = (a: FormArray): void => {
```

Comment:
I'm not a fan to use single letter as obj reference, but if we amend it we could rename it to something meaningful, formArray?

Reply:
This change was not introduced as part of the "retain hidden fields" work, but I can rename it to `formArray`.

Reply:
I can do the same for `handleFormControl` and `handleFormGroup` functions.